docs/admin/admin-search.md

```diff
@@ -12,8 +12,8 @@ documentation, but if you're looking for an example of how it's being used on
 Forem, we've implemented it to help searching and sorting user reports.
 
 The view responsible for managing user reports can be found at
-`localhost:3000/admin/reports` and Ransack can be seen in use on the index
-action of the [`admin/feedback_messages_controller`][feedback_messages].
+`localhost:3000/admin/moderation/reports` and Ransack can be seen in use on the
```

Comment:
tiny nitpick: for consistency we should probably remove "localhost:3000", I see other pages have just the path

```suggestion
`/admin/moderation/reports` and Ransack can be seen in use on the
```